main.md

```diff
@@ -315,18 +334,28 @@ If a client uses `authorization_details` with `locations` elements and the `reso
 
 The authorization server will consider this audience restriction in the user consent if needed.
 
-### Token Request
-
-If  a client uses the `resource` parameter in a token requests, the AS MUST utilize the data provided in the `locations` elements to filter the authorization data objects applicable to the respective resource(s). 
-
-The logic is as follows:
+In the following example (line breaks for illustrative purposes only), 
 
-* For every authorization details object without a `locations` element: the authorization server treats it as applicable to all resources, i.e. it assigns this authorization details object to the access token. 
-* For every authorization details object with a `locations` element: the authorization server adds this object to the access token, if at least one of the `locations` values exactly matches the `resource` parameter value. The authorization server MUST compare both values using an exact byte match of the string values.
-
-This shall be illustrated using an example. 
+```
+GET /authorize?response_type=code
+   &client_id=s6BhdRkqt3
+   &state=af0ifjsldkj
+   &redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb
+   &code_challenge_method=S256
+   &code_challenge=K2-ltc83acc4h0c9w6ESC_rEMTJ3bwc-uCHaoeK1t8U
+   &resource=https%3A%2F%2Fexample%2Ecom%2Fpayments
+   &authorization_details=%5B%7B%22type%22%3A%22account%5Finformation%22%2C%22actions
+   %22%3A%5B%22list%5Faccounts%22%2C%22read%5Fbalances%22%2C%22read%5Ftransactions%22
+   %5D%2C%22locations%22%3A%5B%22https%3A%2F%2Fexample%2Ecom%2Faccounts%22%5D%7D%2C%7
+   B%22type%22%3A%22payment%5Finitiation%22%2C%22actions%22%3A%5B%22initiate%22%2C%22
+   status%22%2C%22cancel%22%5D%2C%22instructedAmount%22%3A%7B%22currency%22%3A%22EUR%
+   22%2C%22amount%22%3A%22123%2E50%22%7D%2C%22creditorName%22%3A%22Merchant123%22%2C%
+   2creditorAccount%22%3A%7B%22iban%22%3A%22DE02100100109307118603%22%7D%2C%22remitta
+   nceInformationUnstructured%22%3A%22RefNumberMerchant%22%7D%5D HTTP/1.1
```

Comment:
Not sure what's going on here but this seems to decode into some JSON that isn't totally valid.

Reply:
Good catch! A quotation mark had been replaced by a comma (I don't know why). Fixed